.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "CellSeg"]
+	path = CellSeg
```

Comment:
Does segmentation happen prior? Is it worth using a different segmentation approach than the one we are using for traditional CellProfiler features? This might cause some unnecessary conflicts when comparing approaches. (FWIW, we only used CellSeg with Dayna's project because it works well with spatial data. We've also been burned in the past when trying to compare approaches with different segmentations.